main.js

```diff
@@ -180,3 +180,15 @@ button4.addEventListener('click', () => {
 
 popupOverlay.addEventListener('click', closePopUp);
 btnClosePop.addEventListener('click', closePopUp);
+
```

Comment:
[OPTIONAL] - I know that this is completely not related to the current milestone. However, I think it can make your contact form more usable.

I noticed that your `submit` button is right at the bottom of the page, which can affect the user experience a little bit.
![image](https://user-images.githubusercontent.com/38088818/163629047-31f0684c-8aa1-40bc-b2bd-3b2f932ae008.png)

I think it would be a great idea to give either the `submit button` or the `form` a `margin-bottom` in order to have some space before the end of the page so we can have something like this.
![image](https://user-images.githubusercontent.com/38088818/163629360-2c704faa-411d-4f44-b191-edaccf05f294.png)
Ultimately, improving the user experience😊 

Otherwise, fantastic job!!! 👏🏾
Your portfolio looks amazing😍